packages/rich-text/src/Toolbar/components/EmbedEntityWidget.tsx

```diff
@@ -31,54 +31,31 @@ export const EmbedEntityWidget = ({ isDisabled, canInsertBlocks }: EmbedEntityWi
   const blockAssetEmbedEnabled =
     isNodeTypeEnabled(sdk.field, BLOCKS.EMBEDDED_ASSET) && canInsertBlocks;
 
-  const numEnabledEmbeds = [
```

Comment:
These changes were already part of https://github.com/contentful/field-editors/pull/1133